src/components/vacation-requests-table/vacation-requests-table-toolbar/toolbar-delete-button.tsx

```diff
@@ -0,0 +1,32 @@
+import { Delete } from "@mui/icons-material";
+import { Button, Typography } from "@mui/material";
+import strings from "../../../localization/strings";
+
+/**
+ * Component properties
+ */
+interface Props {
+  setConfirmationHandlerOpen: (confirmation: boolean) => void;
+}
+
+/**
+ * Delete Button component
+ *
+ * @param props component properties
+ */
+const ToolbarDeleteButton = ({ setConfirmationHandlerOpen }: Props) => (
+  <Button
+    variant="contained"
+    sx={{
+      width: "100%"
+    }}
+    onClick={() => {
+      setConfirmationHandlerOpen(true);
+    }}
+  >
+    <Delete />
+    <Typography variant="h6">&nbsp;{strings.tableToolbar.delete}</Typography>
```

Comment:
Maybe the desired effect could be achieved with styling rather than the nbsp?

Reply:
Removed non-breaking space and added styling to Delete Icon:
`<Delete sx={{ marginRight: 1 }} />
 <Typography variant="h6">{strings.tableToolbar.delete}</Typography>`